source/intel.js

```diff
@@ -2,22 +2,16 @@
 
 const sat_version = "2.21";
 
-const stripHtml = (html) => {
-  let tmp = document.createElement("DIV");
-  tmp.innerHTML = html;
-  return tmp.textContent || tmp.innerText || "";
-};
 
 const image_url = (str) => {
-  let safe_str = stripHtml(str);
   const protocol = "^(https://)";
   const domains = "(i.ibb.co/|i.imgur.com/)";
   const content = "([-#/;&_\\w]{1,150})";
   const images =
     "(.)(gif|jpe?g|tiff?|png|webp|bmp|GIF|JPE?G|TIFF?|PNG|WEBP|BMP)$";
   let regex = new RegExp(protocol + domains + content + images);
   let unused = "foo";
```

Comment:
This can be deleted. It must have been a string I was using for development. lol